src/main/settings.js

```diff
@@ -21,7 +21,8 @@ export const defaultSettings = {
     exitOnWindowClose: true,
     themePrimaryColor: '#7e57c2',
     themeSecondaryColor: '#ff4081',
-    themeVariety: 'light'
+    themeVariety: 'light',
+    autoAddToPlaylist: 'false'
```

Comment:
```suggestion
    autoAddToPlaylist: false
```

This should be a boolean `false`, otherwise

```js
if (this.autoAddToPlaylist) {
    ...
}
```

would always be `true`.

**add to**  playlist seems a bit confusing, would it be better to use **replace** in this case? Because **add** usually means to do something like **append**, but it's replacing current playlist here.

Reply:
My fault, 😆, I've never systematically studied JavaScript..